tests/tools/test_builtin.py

```
@@ -288,20 +291,20 @@ def test_verify_pass(minmax_project, caplog):
 
 def test_verify_fail(minmax_project, caplog):
     project = minmax_project(VerifyTask, parallel=1)
-    project.set('flowgraph', "test", 'end', '0', 'args', 'errors==1041')
+    project.set('flowgraph', "test", 'end', '0', 'args', 'metric0==1041')
 
     node = SchedulerNode(project, "end", "0")
     with node.runtime():
-        with pytest.raises(ValueError, match="^end/0 fails 'errors' metric: 1042==1041$"):
+        with pytest.raises(ValueError, match="^end/0 fails 'metric0' metric: 1042==1041$"):
```

Ruff (0.14.13)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)